### PR TITLE
tartube: 2.0.016 -> 2.1.0

### PR DESCRIPTION
**Pull request by luc65r**
###### Motivation for this change
I updated Tartube and added playsound which is a new dependency.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by luc65r** [comment]
Even though there is `gobject-introspection` in `nativeBuildInputs` and `pygobject3` in `propagatedBuildInputs`, it can't find `gi` when it runs setup.py.
It launches and can download videos.

File: pkgs/development/python-modules/playsound/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "TaylorSMarks";
+    repo = "playsound";
+    rev = "907f1fe73375a2156f7e0900c4b42c0a60fa1d00";
+    sha256 = "1fh3m115h0c57lj2pfhhqhmsh5awzblb7csi1xc5a6f6slhl059k";
+  };
+
+  doCheck = false;
```
Comment (jonringer) [inline pkgs/development/python-modules/playsound/default.nix:17]: Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

Comment (luc65r) [reply pkgs/development/python-modules/playsound/default.nix:17]: playsound provides no tests, and it is just one file (plus setup.py), so I have no idea how I could make checks.

Comment (jonringer) [reply pkgs/development/python-modules/playsound/default.nix:17]: then my first paragraph you can disregard, second still applies
```
pythonImportsCheck = [ "playsound" ];
```

**Comment by Ekleog** [comment]
Thankyou for this PR! :D

**Pull request merged**